Update HttpPusher to support multiple webhooks

s0ph0s-dog opened the pull request:
Someone in IRC suggested that we have the pinned message in Telegram also say the current track. This change enables that ability by having multiple webhooks in the HttpPusher. It also required changing the config format to TOML from INI.

s0ph0s-2 committed:
Update HttpPusher to support multiple webhooks

s0ph0s-2 commented on gelo.toml.example:
```diff
@@ -51,178 +51,195 @@ prefix_file = /tmp/prefix.txt
 # A floating-point number of seconds to delay announcements for.  Useful for
 # compensating for streaming delays from Soma.FM to you.
-extra_delay = 1
+extra_delay = "1"
```
Number strings are gross, this should be fixed.

s0ph0s-2 replied:
c1ccb5a fixes this!

s0ph0s-2 committed:
TOML config is a breaking change

s0ph0s-2 committed:
Use TOML features to simplify config

Every plugin that previously used non-string types (bool, int, float) for configuration values now relies on TOML to parse them correctly. All configuration validation checks now check the type of the value, rather than the result of a conversion attempt.

Validation for repeat_with in the IRC plugin now also ensures that the key, if present, is an array containing only strings.
